Formula/platformio.rb

```diff
@@ -3,8 +3,8 @@ class Platformio < Formula
 
   desc "An open-source ecosystem for embedded development"
   homepage "https://platformio.org/"
-  url "https://files.pythonhosted.org/packages/f4/c8/8e472c22602ec6cdc6f93e35357ff5f3dff3434da6fc76cd3cd57c7eefb6/platformio-4.2.1.tar.gz"
-  sha256 "310fa8b624cf938ef95284ae3a40a9caa36aafef32ac30ad588b43df516945c3"
+  url "https://files.pythonhosted.org/packages/0d/cd/6e7a07fa7bf9c52f7d69572f17fd58feee74013de03519698b44b4735d85/platformio-4.3.0.tar.gz"
+  sha256 "27a34001a622fdf6478562f08d15d5624b11bc702343435802a8dfe6526defcd"
   revision 1
```

Comment:
Remove this line please.

Reply:
Every time I remove this line and Homebrew bot adds it again. Why?

Reply:
It's not added by test-bot.

See: https://github.com/Homebrew/homebrew-core/commit/2be0f45aa54dd5dd45155d91b80221ffaf52d8ed

Reply:
Thanks! What does mean that commit by @Bo98?